cmd/sponged/handlers/item.go

```diff
@@ -101,7 +101,8 @@ func (itemHandle) Upsert(c *app.Context) error {
 		return err
 	}
 
-	c.Respond(nil, http.StatusNoContent)
+	c.Respond(itMap, http.StatusOK)
```

Comment:
When I say the variable I say  it  or  it map.  I think the context is being lost.